tests/integration/tests/conftest.py

```diff
@@ -51,7 +51,7 @@ def pytest_configure(config):
         "markers",
         "node_count: Mark a test to specify how many instance nodes need to be created\n"
         "disable_k8s_bootstrapping: By default, the first k8s node is bootstrapped. This marker disables that."
-        "etcd_count: Mark a test to specify how many etcd instance nodes need to be created (None by default)",
+        "etcd_count: Mark a test to specify how many etcd instance nodes need to be created (None by default)\n",
```

Comment:
what is this about?

Reply:
Drive-by cosmetic fix. This would otherwise be displayed on the same line (followed the `node_count` lead here)